operator/charts/kit-operator/crds/control-plane-crd.yaml

```diff
@@ -12,6 +12,8 @@ spec:
     listKind: ControlPlaneList
```

Comment:
These files are auto generated by `hack/codegen.sh` we shouldn't be making changes directly to these files. 

We should add the `kubebuilder` [tags for ControlPlane](https://github.com/awslabs/kubernetes-iteration-toolkit/blob/main/operator/pkg/apis/controlplane/v1alpha1/controlplane.go#L24) and DataPlane object definitions for shortname and run `make codegen` that will update these files.